### PR TITLE
Array binary search

### PR DESCRIPTION
**Pull request by Gr8-Dayne**

### CODE REVIEW

**Comment by Gr8-Dayne** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

File: challenges/src/test/java/challenges/BinarySearchTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    @Test public void testBinaryStuff() {
+
+        int[] arr = {1, 2, 3, 4, 5, 6};
+        int key = 1;
+
+        int actualOutput = BinarySearch.BinarySearcher(arr, key);
+
+        System.out.println("actualOutput = " + actualOutput);
+
```
Comment (ncarignan) [inline challenges/src/test/java/challenges/BinarySearchTest.java]: it looks like you are trying to find out what to test here. It is correctly outputting that the position of value 1 is 0, your test can check assertEquals(0, actualOutput)

Comment (Gr8-Dayne) [reply challenges/src/test/java/challenges/BinarySearchTest.java]: Thank you for pointing that out, super bone-head mistake on my part.
